noir-projects/aztec-nr/aztec/src/macros/functions/utils.nr

```diff
@@ -233,19 +233,19 @@ pub(crate) comptime fn transform_public(f: FunctionDefinition) {
         };
     };
 
-    let name = f.name();
+    let function_name = f.name();
+
     // Modifications introduced by the different marker attributes.
-    let internal_check = if is_fn_internal(f) {
-        let assertion_message = f"Function {name} can only be called internally";
+    let internal_check = if is_fn_only_self(f) {
+        let assertion_message = f"Function {function_name} can only be called by the same contract";
         quote { assert(self.msg_sender().unwrap() == self.address, $assertion_message); }
     } else {
         quote {}
     };
 
     let view_check = if is_fn_view(f) {
-        let name = f.name();
         let assertion_message =
-            f"Function {name} can only be called statically".as_ctstring().as_quoted_str();
+            f"Function {function_name} can only be called statically".as_ctstring().as_quoted_str();
         quote { assert(self.context.is_static_call(), $assertion_message); }
```

Comment:
Unrelated change. This is just to keep the style in line with `transform_private`.